### ISSUE TITLE
Background mode not supported in LM Studio (client.responses.create)

### ISSUE THREAD

**Issue by MeetSolanki530** [lmstudio-ai/lmstudio-bug-tracker#1193]
### **Bug Report Example**

**LM Studio Version:**
0.3.31(Build 7)

**Operating System:**
Windows 11

**Bug Description:**
Running `client.responses.create(..., background=True)` causes the request to hang indefinitely. The `background` parameter appears unsupported for local models.

**To Reproduce:**

1. Launch LM Studio and load `qwen3-vl-2b-instruct`.
2. Run the following code:

   ```python
   resp = client.responses.create(
       model="qwen3-vl-2b-instruct",
       input="Write a very long novel about otters in space.",
       background=True,
   )
   print(resp.status)
   ```
3. The program needs to run in background as openai responses api docs.

**Expected Behavior:**
Either an error message explaining that background mode isn’t supported, or proper async task creation as per openai docs https://platform.openai.com/docs/guides/background#polling-background-responses.

**Actual Behavior:**
The call never completes.

**Logs / Screenshots:**
AS Per OpenAI Responses API Doc there needs Background Mode.

2025-11-05 22:01:08 [DEBUG]
 Received request: POST to /v1/responses with body  {
  "background": true,
  "input": "Write a very long novel about otters in space.",
  "model": "qwen3-vl-2b-instruct"
}
2025-11-05 22:01:08  [INFO]
 [qwen3-vl-2b-instruct] Generating synchronous v1/responses response...
2025-11-05 22:01:08 [DEBUG]
 Sampling params:	repeat_last_n = 64, repeat_penalty = 1.100, frequency_penalty = 0.000, presence_penalty = 0.000
	dry_multiplier = 0.000, dry_base = 1.750, dry_allowed_length = 2, dry_penalty_last_n = -1
	top_k = 40, top_p = 0.950, min_p = 0.050, xtc_probability = 0.000, xtc_threshold = 0.100, typical_p = 1.000, top_n_sigma = -1.000, temp = 0.100
	mirostat = 0, mirostat_lr = 0.100, mirostat_ent = 5.000
Sampling: 
logits -> logit-bias -> penalties -> dry -> top-n-sigma -> top-k -> typical -> top-p -> min-p -> xtc -> temp-ext -> dist 
Generate: n_ctx = 4096, n_batch = 512, n_predict = -1, n_keep = 19
Cache reuse summary: 19/19 of prompt (100%), 19 prefix, 0 non-prefix
2025-11-05 22:01:08  [INFO]
 [qwen3-vl-2b-instruct] Prompt processing progress: 0.0%
2025-11-05 22:01:08  [INFO]
 [qwen3-vl-2b-instruct] Prompt processing progress: 100.0%
2025-11-05 22:01:08 [DEBUG]
 Total prompt tokens: 19
Prompt tokens to decode: 1
BeginProcessingPrompt
FinishedProcessingPrompt. Progress: 100
2025-11-05 22:01:58 [DEBUG]
 [lmstudio-llama-cpp] Error in predictTokens: Reached context length of 4096 tokens with model (arch: qwen3vl) that does not currently support mid-generation context overflow. Try reloading with a larger context length or shortening the prompt/chat.
2025-11-05 22:01:58 [DEBUG]
 1 Error predicting: _0x2eaace [Error]: Reached context length of 4096 tokens with model (arch: qwen3vl) that does not currently support mid-generation context overflow. Try reloading with a larger context length or shortening the prompt/chat.
    at _0x31d8dd.<computed>.predictTokens (C:\Users\solan_k4hy1rm\AppData\Local\Programs\LM Studio\resources\app\.webpack\lib\llmworker.js:85:29916)
    at async _0xea65e0.predictTokens (C:\Users\solan_k4hy1rm\AppData\Local\Programs\LM Studio\resources\app\.webpack\lib\llmworker.js:113:17922)
    at async _0xea65e0.handleMessage (C:\Users\solan_k4hy1rm\AppData\Local\Programs\LM Studio\resources\app\.webpack\lib\llmworker.js:113:7122) {
  cause: undefined,
  suggestion: undefined,
  errorData: undefined,
  data: undefined,
  displayData: undefined,
  title: 'Reached context length of 4096 tokens with model (arch: qwen3vl) that does not currently support mid-generation context overflow. Try reloading with a larger context length or shortening the prompt/chat.'
}
2025-11-05 22:01:59 [DEBUG]
 Received request: POST to /v1/responses with body  {
  "background": true,
  "input": "Write a very long novel about otters in space.",
  "model": "qwen3-vl-2b-instruct"
}
2025-11-05 22:01:59  [INFO]
 [qwen3-vl-2b-instruct] Generating synchronous v1/responses response...
2025-11-05 22:01:59 [DEBUG]
 Sampling params:	repeat_last_n = 64, repeat_penalty = 1.100, frequency_penalty = 0.000, presence_penalty = 0.000
	dry_multiplier = 0.000, dry_base = 1.750, dry_allowed_length = 2, dry_penalty_last_n = -1
	top_k = 40, top_p = 0.950, min_p = 0.050, xtc_probability = 0.000, xtc_threshold = 0.100, typical_p = 1.000, top_n_sigma = -1.000, temp = 0.100
	mirostat = 0, mirostat_lr = 0.100, mirostat_ent = 5.000
Sampling: 
logits -> logit-bias -> penalties -> dry -> top-n-sigma -> top-k -> typical -> top-p -> min-p -> xtc -> temp-ext -> dist
2025-11-05 22:01:59 [DEBUG]
 Generate: n_ctx = 4096, n_batch = 512, n_predict = -1, n_keep = 19
Cache reuse summary: 19/19 of prompt (100%), 19 prefix, 0 non-prefix
Total prompt tokens: 19
Prompt tokens to decode: 1
BeginProcessingPrompt
FinishedProcessingPrompt. Progress: 100
2025-11-05 22:01:59  [INFO]
 [qwen3-vl-2b-instruct] Prompt processing progress: 0.0%
2025-11-05 22:01:59  [INFO]
 [qwen3-vl-2b-instruct] Prompt processing progress: 100.0%
2025-11-05 22:02:51 [DEBUG]
 [lmstudio-llama-cpp] Error in predictTokens: Reached context length of 4096 tokens with model (arch: qwen3vl) that does not currently support mid-generation context overflow. Try reloading with a larger context length or shortening the prompt/chat.
2025-11-05 22:02:51 [DEBUG]
 1 Error predicting: _0x2eaace [Error]: Reached context length of 4096 tokens with model (arch: qwen3vl) that does not currently support mid-generation context overflow. Try reloading with a larger context length or shortening the prompt/chat.
    at _0x31d8dd.<computed>.predictTokens (C:\Users\solan_k4hy1rm\AppData\Local\Programs\LM Studio\resources\app\.webpack\lib\llmworker.js:85:29916)
    at async _0xea65e0.predictTokens (C:\Users\solan_k4hy1rm\AppData\Local\Programs\LM Studio\resources\app\.webpack\lib\llmworker.js:113:17922)
    at async _0xea65e0.handleMessage (C:\Users\solan_k4hy1rm\AppData\Local\Programs\LM Studio\resources\app\.webpack\lib\llmworker.js:113:7122) {
  cause: undefined,
  suggestion: undefined,
  errorData: undefined,
  data: undefined,
  displayData: undefined,
  title: 'Reached context length of 4096 tokens with model (arch: qwen3vl) that does not currently support mid-generation context overflow. Try reloading with a larger context length or shortening the prompt/chat.'
}
